packages/core/src/mpi/get-patient-by-demo.ts

```diff
@@ -60,7 +61,9 @@ export const getPatientByDemo = async ({
 
   if (matchingPatients.length > 1) {
     log(
-      `WARNING: matchPatients in getPatientByDemo for demo: ${demo} and cxId: ${cxId} is returning more than one matched patient!`
+      `WARNING: matchPatients in getPatientByDemo for demo: ${JSON.stringify(
+        demo
+      )} and cxId: ${cxId} is returning more than one matched patient!`
```

Comment:
nit: when going through messages to diagnose something its usually helpful to have the mor abstract context to the left and the details to the right, as well as trying to keep messages small - usually there's a limit on columns displayed, requiring one to click to see details. I suggest we leave the details to the end of the message, like:

```ts
log(`WARN: matchPatients returned more than one patient for cxId: ${cxId} and demo: ${JSON.stringify(demo)}`);
```

With that, one can also use the same message on `capture.xxx()`:

```ts
const msg = `matchPatients returned more than one patient`;
log(`WARN: ${msg} for cxId: ${cxId} and demo: ${JSON.stringify(demo)}`);
capture.message(msg, {...});
```

Reply:
Good call. Done!